packages/create-bison-app/template/components/CenteredBoxForm.tsx

```diff
@@ -10,13 +10,11 @@ export function CenteredBoxForm({ children }: Props) {
   return (
     <Box
       margin="auto"
-      borderRadius="md"
+      borderRadius="xl"
       width={{ base: 'auto', lg: 400 }}
-      boxShadow="md"
-      borderColor="gray.200"
-      borderWidth={1}
+      boxShadow={{ base: 'none', sm: useColorModeValue('md', 'md-dark') }}
       padding={8}
-      bg="white"
+      bg={useColorModeValue('white', 'gray.800')}
```

Comment:
Can we try a semantic token value like you did with `muted` below?

```
semanticTokens: {
    colors: {
      'chakra-body-text': [Object],
      'chakra-body-bg': [Object],
      'chakra-border-color': [Object],
      'chakra-placeholder-color': [Object],
      'bg-canvas': [Object],
      'bg-surface': [Object],
      'bg-subtle': [Object],
      'bg-muted': [Object],
      default: [Object],
      inverted: [Object],
      emphasized: [Object],
      muted: [Object],
      subtle: [Object],
      border: [Object],
      accent: [Object],
      success: [Object],
      error: [Object],
      'bg-accent': 'brand.600',
      'bg-accent-subtle': 'brand.500',
      'bg-accent-muted': 'brand.400',
      'on-accent': 'white',
      'on-accent-muted': 'brand.50',
      'on-accent-subtle': 'brand.100'
    }
  },
```

Perhaps bg-surface or bg-canvas yields a similar result and avoids adding the hook?

Reply:
Oh yeah, good catch -- `bg-surface` worked great 

I tried to see if I was able to achieve the same with `boxShadow` but that one is a bit tricky since the form falls flat on dark mode.

Reply:
Just an FYI: 
Dark mode reads: https://css-tricks.com/a-complete-guide-to-dark-mode-on-the-web/#aa-dark-mode-shadows